keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -596,29 +585,23 @@ https://storage.googleapis.com/k8s-triage/index.html
 There will be added tests to verify:
 
 - API servers using the old and new allocators at same time
-- API server upgrade from old to new allocatr
+- API server upgrade from old to new allocator
 - ServicesCIDRs resizing
 - ServiceCIDR without the IPv6 limitation size
+- Default ServiceCIDR can be migrated to a new one
+- ServiceCIDR can not leave orhpan IPAddress objects
 
 Files:
 
 - test/integration/controlplane/synthetic_controlplane_test.go
 - test/integration/servicecidr/allocator_test.go
-- test/integration/servicecidr/main_test.go
+- test/integration/servicecidr/migration_test.go
+- test/integration/servicecidr/servicecidr_test.go
 
 ##### e2e tests
 
-<!--
-This question should be filled when targeting a release.
-For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
-
-For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
-https://storage.googleapis.com/k8s-triage/index.html
-
-We expect no non-infra related flakes in the last month as a GA graduation criteria.
--->
-
-e2e tests will cover all the user stories defined in the KEP
+e2e tests will validate that components that depend on Services are able
+to use the new ServiceCIDR ranges allocated.
```

Comment:
If we have e2e which were written during alpha stage as mentioned in the graduation criteria, do we have results from running them we could link here?

Reply:
I have in mind this was running in some job, but we went through a lot of refactors with alphas and betas and I can't find them, I'm setting up for sig network only alpha and beta features that will run it https://github.com/kubernetes/test-infra/pull/32739